README.md

```diff
@@ -147,46 +147,131 @@ Graphjet includes [CLICK, FAVORITE, RETWEET, REPLY, AND TWEET as input node type
+- “The current model was last trained several years ago, and uses some very strange features. We are working on training a new model, and eventually rebuilding this part of the stack entirely.”
+
+Twitter has separate models for ranking in-network and out-network tweets, with different features
+- [In-network model](https://github.com/twitter/the-algorithm/blob/main/src/python/twitter/deepbird/projects/timelines/configs/recap_earlybird/feature_config.py)
+- [Out-of-network model](https://github.com/twitter/the-algorithm/blob/main/src/python/twitter/deepbird/projects/timelines/configs/rectweet_earlybird/feature_config.py)
+
+
+Things we would like to find out
+- Since the model is a logistic regression model, what are the weights for each item?
+- How many items will remain after the light ranker?
+
+
+### Heavy Ranker
+
 + **Recap** The "Heavy Ranker" is a [parallel masknet](https://arxiv.org/abs/2102.07619). Majority of the code for this is in the [ML repo](https://github.com/twitter/the-algorithm-ml/blob/main/projects/home/recap/README.md). The [ranker itself](https://github.com/twitter/the-algorithm-ml/blob/main/projects/home/recap/README.md) is run after the candidate generators. 
 
 [Input features](https://github.com/twitter/the-algorithm-ml/blob/main/projects/home/recap/FEATURES.md) 
 
-Outputs are predictions on how user will respond to the tweet: 
-+ probability the user will favorite the Tweet
-+ probability the user will click into the conversation of this tweet and reply or like a Tweet
-+ probability the user will click into the conversation of this Tweet and stay there for at least 2 minutes.
-+probability the user will react negatively (requesting "show less often" on the Tweet or author, block or mute the Tweet author) 
-+ probability the user opens the Tweet author profile and Likes or replies to a Tweet
-+ probability the user replies to the Tweet
-+ probability the user replies to the Tweet and this reply is engaged by the Tweet author 
-+ probability the user will click Report Tweet 
-+ probability the user will ReTweet the Tweet
-+ probability (for a video Tweet) that the user will watch at least half of the video
-
 All of these are combined and weighted into a score. Hyperparameters for the model [and weighting are here.](https://github.com/twitter/the-algorithm-ml/blob/78c3235eee5b4e111ccacb7d48e80eca019e480c/projects/home/recap/config/local_prod.yaml#L1)
 
 For more details on the model, see the [Architecture overview](masknet.md).
 
-+ The Light Ranker
+
+### Scoring Plan
+
+After the model predicts the probability of the actions, weights are assigned to the probability. The tweet with the highest score is likely to appear at the top of your feed.
+
+These are the actions predicted, and [their corresponding weights](https://raw.githubusercontent.com/twitter/the-algorithm-ml/main/projects/home/recap/README.md)
++ probability the user will favorite the Tweet (0.5)
```

Comment:
if you have time, it could be nice to format these into a table so it's a bit easier to read 

https://www.tablesgenerator.com/markdown_tables

```
| feature                                                                          | weight |
|----------------------------------------------------------------------------------|--------|
| p(user user will favorite the Tweet)                                             | .5     |
| p(user will click into the conversation of this tweet and reply or like a Tweet) | 11     |
|                                                                                  |        |
|                                                                                  |        |
```